src/assets/TokenRatesController.test.ts

```diff
@@ -5,10 +5,12 @@ import { NetworkController } from '../network/NetworkController';
 import { TokenRatesController } from './TokenRatesController';
 import { TokensController } from './TokensController';
 
-const COINGECKO_HOST = 'https://api.coingecko.com';
-const COINGECKO_ETH_PATH = '/api/v3/simple/token_price/ethereum';
-const COINGECKO_BSC_PATH = '/api/v3/simple/token_price/binance-smart-chain';
-const COINGECKO_ASSETS_PATH = '/api/v3/asset_platforms';
+const COINGECKO_HOST = 'https://api.coingecko.com/api/v3';
```

Comment:
Nit: Expanding this to include the base API path is fine, but now this variable name seems misleading. Maybe `COINGECKO_API` would be better?